Source/WebCore/layout/integration/inline/InlineIteratorInlineBox.h

```diff
@@ -83,6 +84,11 @@ inline InlineBoxIterator InlineBox::iterator() const
     return { *this };
 }
 
+inline bool InlineBox::isSingular() const
+{
+    return !nextInlineBoxLineRightward() && !nextInlineBoxLineLeftward();
+}
+
```

Comment:
Wasn't sure what to name it, this seemed reasonable, but open to other ideas.

Reply:
non-spanning is what we have in IFC but I should not be making comments about naming things.

Reply:
Gecko would probably call it something like `hasContinuation`...

Reply:
that would clash with our definition of continuation (which should hopefully go away sometime soon)

Reply:
though I still prefer it over "singular"

Reply:
Oh, what about `isSplit()` (returning the inverse)?

Reply:
that sounds reasonable